.github/workflows/ci.yml

```diff
@@ -1,17 +1,14 @@
-name: '@ngneat/reactive-forms'
+name: 'lothern/reactive-forms'
 
 on:
+  workflow_dispatch:
```

Comment:
The entire code uses lothern?

Reply:
Sorry, those changes weren't supposed to make it into the master branch. I've corrected it.